From c978b9f8b2d96d9f4dd45be4e6f44491efcd14ca Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Wed, 10 Nov 2021 22:08:25 +0100
Subject: [PATCH] Patch to inject Tactician into the standard library

---
 theories/Init/Ltac.v | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/theories/Init/Ltac.v b/theories/Init/Ltac.v
index ac5a69a38a..df188a5af7 100644
--- a/theories/Init/Ltac.v
+++ b/theories/Init/Ltac.v
@@ -8,6 +8,5 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-Declare ML Module "ltac_plugin".
-
-Export Set Default Proof Mode "Classic".
+From Tactician Require Import Ltac1.Record.
+Export Set Default Proof Mode "Tactician Ltac1".
-- 
2.34.0

