From 8fb6d2489f9a979cf3f0502441b75c5c45d9e5d5 Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Wed, 10 Nov 2021 21:45:29 +0100
Subject: [PATCH] Patch to inject Tactician into the standard library

---
 theories/Init/Notations.v | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/theories/Init/Notations.v b/theories/Init/Notations.v
index 77236f77df..eb27928c26 100644
--- a/theories/Init/Notations.v
+++ b/theories/Init/Notations.v
@@ -125,6 +125,5 @@ Open Scope type_scope.
 
 (** ML Tactic Notations *)
 
-Declare ML Module "ltac_plugin".
-
-Global Set Default Proof Mode "Classic".
+From Tactician Require Import Ltac1.Record.
+Global Set Default Proof Mode "Tactician Ltac1".
-- 
2.34.0

