From 1567301f73d0b6f3968ccea9df9783dfc624cc31 Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Wed, 10 Nov 2021 22:08:25 +0100
Subject: [PATCH] Patch to inject Tactician into the standard library

---
 theories/Init/Ltac.v | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/theories/Init/Ltac.v b/theories/Init/Ltac.v
index ac5a69a38a..87a38e4851 100644
--- a/theories/Init/Ltac.v
+++ b/theories/Init/Ltac.v
@@ -11,3 +11,6 @@
 Declare ML Module "ltac_plugin".
 
 Export Set Default Proof Mode "Classic".
+
+From Tactician Require Import Ltac1.Record.
+Export Set Default Proof Mode "Tactician Ltac1".
>>>>>>> coq8.12
-- 
2.33.1

