From efc6af59b5e4edaf3e98072ad8a46b23c1da1559 Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Wed, 10 Nov 2021 21:45:29 +0100
Subject: [PATCH] Patch to inject Tactician into the standard library

---
 theories/Init/Notations.v | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/theories/Init/Notations.v b/theories/Init/Notations.v
index 77236f77df..7c56963295 100644
--- a/theories/Init/Notations.v
+++ b/theories/Init/Notations.v
@@ -125,6 +125,5 @@ Open Scope type_scope.
 
 (** ML Tactic Notations *)
 
-Declare ML Module "ltac_plugin".
-
-Global Set Default Proof Mode "Classic".
+From Tactician Require Export Ltac1.Record.
+Global Set Default Proof Mode "Tactician Ltac1".
-- 
2.34.0

