From a6fc663a94027bd28ffabe3165e5d36de40c6134 Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Wed, 10 Nov 2021 21:45:29 +0100
Subject: [PATCH] Patch to inject Tactician into the standard library

---
 theories/Init/Notations.v | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/theories/Init/Notations.v b/theories/Init/Notations.v
index 77236f77df..81f93cab4e 100644
--- a/theories/Init/Notations.v
+++ b/theories/Init/Notations.v
@@ -128,3 +128,6 @@ Open Scope type_scope.
 Declare ML Module "ltac_plugin".
 
 Global Set Default Proof Mode "Classic".
+
+From Tactician Require Import Ltac1.Record.
+Global Set Default Proof Mode "Tactician Ltac1".
-- 
2.33.1

